draft-ietf-tls-tls13.md

```diff
@@ -3324,11 +3353,33 @@ because TLS does not directly use this secret for anything
 other than for computing other secrets.)
 
 
-#  Mandatory Cipher Suites
+#  Mandatory Algorithms
+
+##  MTI Cipher Suites
 
 In the absence of an application profile standard specifying otherwise, a
 TLS-compliant application MUST implement the cipher suite [TODO:Needs to be selected](https://github.com/tlswg/tls13-spec/issues/32). (See {{cipher-suites}} for the definition.)
 
+##  MTI Extensions
+
+In the absence of an application profile standard specifying otherwise, a
+TLS-compliant application MUST implement the following TLS extensions:
+
+  * Server Name Indication ("server_name"; Section 3 of {{RFC6066}})
+  * Signature Algorithms ("signature_algorithms"; {{signature-algorithms}})
+  * Negotiated Groups ("supported_groups"; {{negotiated-groups}})
+
+All implementations MUST support usage of the "server_name" extension with
+applications capable of using it. All implementations MUST use the
+"signature_algorithms" extension when offering and negotiating certificate
+authenticated cipher suites. All implementations MUST use the
+"supported_groups" extension when offering and negotiating DHE or ECDHE
+cipher suites.
+
+Additionally, any implementations that allow handshake renegotiation in prior
+versions of TLS MUST implement the "renegotiation_info" extension. When
+negotiating TLS 1.2 or prior, endpoints MUST support and use the
+"renegotiation_info" extension or refuse all renegotiation attempts.
```

Comment:
As noted previously, please remove the requirement to do RI and send SNI. Feel free to start a separate mailing list thread with this topic.